recipes/igl/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "igl" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/geometryprocessing/libigl-python-bindings/archive/{{ version }}.tar.gz
+  sha256: cc749e5742986b20a440672a2a426b2b72598b179b411163768584009f44d445
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
+    - cmake
+
+  host:
+    - python
+    - pip
+    - setuptools
```

Comment:
With `pip` you can remove `setuptools` here.

Reply:
sure, done